### DISCUSSION TITLE
Removing default values from base style

### DISCUSSION THREAD

**Discussion by paul-vd** [tailwindlabs/tailwindcss#3471] [Help] [answered]
Is there a method to remove some styles included in base?

for example my `@import base` is added these styles which is causing unwanted borders and sizes:
![image](https://user-images.githubusercontent.com/39598117/106166796-d57d3380-618c-11eb-9698-275677974f14.png)

here is an example output:  the first and last fields are correctly styled where the 3 middle fields are being styled by the base of tailwind.
![image](https://user-images.githubusercontent.com/39598117/106166963-00678780-618d-11eb-93e6-44ce45384184.png)

**Comment by paul-vd** [comment]
:eyes:

**Comment by adamwathan** [comment, marked as answer]
Those styles are coming from the @tailwindcss/forms plugin. If you don't want those base styles you can remove that plugin, or write some custom base CSS to override it, or override those styles in your HTML with utilities which is what that plugin is designed to support.

**Reply by paul-vd** [reply]
Thanks! now everything is back to normal :blush: I added that  without needing it, and total forgot about it. :man_facepalming:
